code/modules/surgery/organs/organ.dm

```diff
@@ -50,21 +50,18 @@
 /obj/item/organ/proc/update_health()
 	return
 
-/obj/item/organ/New(mob/living/carbon/holder, datum/species/species_override = null)
+/obj/item/organ/New(mob/living/carbon/human/holder, datum/species/species_override = null)
```

Comment:
Are you sure about this? Xenomorphs do have organs

Reply:
Xenos have organs but they don't have DNA